purchase_default_terms_conditions/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "name": "Purchase Default Terms Conditions",
     "summary": """
         This module allows purchase default terms & conditions""",
-    "version": "16.0.1.0.2",
+    "version": "17.0.1.0.2",
```

Comment:
@nabilragab Please bump version to 17.0.1.0.0

Reply:
I squashed this into @nabilragab 's original migration commit, should be fixed now @amkarthik